packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +40,30 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        const commandsUsed = (commands) => {
```

Comment:
This helper doesn't require the `commands` arg to be passed in explicitly, it is already available in the global scope.